check-versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+check_version_in_file() {
+    local VERSION=$1
+    local FILEPATH=$2
+    # shellcheck disable=SC2155
+    local SHELL_FILE_DIR=$(dirname "$0")
+
+    grep -q "$VERSION" "$SHELL_FILE_DIR/$FILEPATH" || { echo "String '$VERSION' not found in '$FILEPATH' or file does not exist."; return 1; }
```

Comment:
I always recommend using expanded command parameters when used in script files for easier to read code/scripts:
```suggestion
    grep --quiet "$VERSION" "$SHELL_FILE_DIR/$FILEPATH" || { echo "String '$VERSION' not found in '$FILEPATH' or file does not exist."; return 1; }
```